reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md

```diff
@@ -32,76 +32,122 @@ Be noted that when using the ZIP archive, you won't get the prerequisites check
+  # space for the zip and the unzipped contents
+  # the path should be local to the device
+  Copy-Item .\PowerShell-6.0.2-win-arm32.zip -Destination u:\users\administrator\Downloads -ToSession $s
+  ```
+
+- Connect to the device and expand the archive
+
+  ```powershell
+  Enter-PSSession $s
+  cd u:\users\administrator\downloads
+  Expand-Archive .\PowerShell-6.0.2-win-arm32.zip
+  ```
+
+- Setup remoting to PowerShell Core 6
+
+  ```powershell
+  cd .\PowerShell-6.0.2-win-arm32
+  # Be sure to use the -PowerShellHome parameter otherwise it'll try to create a new
+  # endpoint with Windows PowerShell 5.1
+  .\Install-PowerShellRemoting.ps1 -PowerShellHome .
+  # You'll get an error message and will be disconnected from the device because it has to restart WinRM
+  ```
+
+- Connect to PowerShell Core 6 endpoint on device
+
+  ```powershell
+  # Be sure to use the -Configuration parameter.  If you omit it, you will connect to Windows PowerShell 5.1
+  Enter-PSSession -ComputerName <deviceIp> -Credential Administrator -Configuration powershell.6.0.2
+  ```
+
 ## Deploying on Nano Server
 
-These instructions assume that a version of PowerShell is already running on the Nano Server image and that it has been generated by the [Nano Server Image Builder](https://technet.microsoft.com/windows-server-docs/get-started/deploy-nano-server).
-Nano Server is a "headless" OS and deployment of PowerShell Core binaries can happen in two different ways:
+These instructions assume that a version of PowerShell is already running on the Nano Server image
+and that it has been generated by the [Nano Server Image Builder](/windows-server/get-started/deploy-nano-server).
+Nano Server is a "headless" OS. Core binaries can be deploy using two different methods.
 
 1. Offline - Mount the Nano Server VHD and unzip the contents of the zip file to your chosen location within the mounted image.
-1. Online - Transfer the zip file over a PowerShell Session and unzip it in your chosen location.
+2. Online - Transfer the zip file over a PowerShell Session and unzip it in your chosen location.
 
 In both cases, you will need the Windows 10 x64 Zip release package and will need to run the commands within an "Administrator" PowerShell instance.
 
 ### Offline Deployment of PowerShell Core
 
 1. Use your favorite zip utility to unzip the package to a directory within the mounted Nano Server image.
-1. Unmount the image and boot it.
-1. Connect to the inbox instance of Windows PowerShell.
-1. Follow the instructions to create a remoting endpoint using the [another instance technique](#executed-by-another-instance-of-powershell-on-behalf-of-the-instance-that-it-will-register).
+2. Unmount the image and boot it.
+3. Connect to the inbox instance of Windows PowerShell.
+4. Follow the instructions to create a remoting endpoint using the [another instance technique](#executed-by-another-instance-of-powershell-on-behalf-of-the-instance-that-it-will-register).
```

Comment:
Use quotes for "another instance technique" like you have below?

Reply:
fixed